source/scripts/app.js

```diff
@@ -751,6 +781,83 @@ function connectRecipeAction() {
   });
 }
 
+/**
+ * Adds an event listener to the "Save Changes" button in the "Edit Cookbook"
+ * page.
+ * @param {String} oldTitle
+ */
+function buttonsEditCookbook() {
+  // Get the "Save Changes" button
+  "use strict";
+  let templatePage = document.querySelector("edit-cookbook");
+  let shadow = templatePage.shadowRoot;
+  let saveButton = shadow
+    .querySelector("div")
+    .children[3].getElementsByTagName("button")[0];
+
+  let cancelButton = shadow
+    .querySelector("div")
+    .children[2].getElementsByTagName("button")[0];
+
+  saveButton.addEventListener("click", async () => {
+    // Get the Title and the Description
+    let templatePage = document.querySelector("edit-cookbook");
+    let shadow = templatePage.shadowRoot;
+    let mainDiv = shadow.querySelector("div.input-container");
+
+    // Gets the div by index (first div = 0, second div = 1)
+    let title = mainDiv.children[0].getElementsByTagName("input")[0].value;
+    let description =
+      mainDiv.children[1].getElementsByTagName("input")[0].value;
+
+    // Nothing to update
+    if (
+      (title === null || title === "") &&
+      (description === null || description === "")
+    ) {
```

Comment:
We can remove the `null`s if we just use `""` instead
```suggestion
    if (title === "" && description === "") {
```